$NetBSD: patch-ad,v 1.1 2009/09/20 12:36:03 sno Exp $

Apply patch from FreeBSD ports - FreeBSD's <net/if.h> doesn't include
sys/socket.h ...

--- src/bytestream-socks5.c.orig	2009-09-20 14:18:46.000000000 +0200
+++ src/bytestream-socks5.c	2009-09-20 14:24:08.000000000 +0200
@@ -21,6 +21,10 @@
 #include "config.h"
 #include "bytestream-socks5.h"
 
+#ifdef __FreeBSD__
+#include <sys/socket.h>
+#include <sys/types.h>
+#endif
 #include <arpa/inet.h>
 #include <errno.h>
 #include <fcntl.h>
